gef-ui/templates/partials/amendments/what-needs-to-change.njk

```diff
@@ -17,7 +17,7 @@
     <span class="govuk-caption-l" data-cy="heading-caption">
       {{ exporterName }}
     </span>
-    <h1 class="govuk-label-wrapper" data-cy="page-title">
+    <h1 class="govuk-label-wrapper" data-cy="page-heading">
```

Comment:
Just for my understanding was title class always wrong?

Reply:
It makes no difference -it's just the testing value. All the other pages in the amendments flow have used 'page-heading' so just changed here to keep consistent 👍